appwrite/appwrite
# Only throw exception on nonzero exit code

## What does this PR do?

Reconfigures the certificates worker to throw an exception only on a non-zero exit code from `certbot`. [This open issue](https://github.com/certbot/certbot/issues/3666) details the current behavior, which includes printing INFO messages to STDERR. 

## Test Plan

- Deploy to VPS
- Configure domain and env vars 
- Confirm 🔐 by navigating to domain with LetsEncrypt certificate.

## Related PRs and Issues

#1001 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.
